app/models/business.rb

```diff
@@ -15,5 +16,9 @@ class Business < ApplicationRecord
   validates :bio, presence: true
   validates :developer_notifications, inclusion: {in: developer_notifications.keys}
 
+  scope :newest_first, -> { order(created_at: :desc) }
+
+  pg_search_scope :filter_by_search_query, against: [:company, :contact_name], associated_against: {user: :email}, using: {tsearch: {prefix: true}}
```

Comment:
What does `prefix: true` do?

Perhaps related, if someone's email is `foo@bar.com` I'd like to be able to search for `bar.com` and see them in the results. I don't think that's currently possible.

Reply:
Yeah, `prefix: true` will allow partial searches from the beginning of a database string (`foo` will give you `foo@bar.com`), but won't allow for search results starting anywhere else in the string (`@bar` won't give you `foo@bar.com`).  

So, in order to allow for searches the way you're describing, I'm assuming that I need to create another virtual column similar to the `textsearchable_index_col` column that already exists, but base the value on name & email instead of the hero and bio.  Is that the right direction to go in?

Reply:
That sounds right to me!

Unless removing `prefix: true` gets us what we want? But it doesn't sound like it will.